src/common/components/ProfileInfoContent.tsx

```diff
@@ -22,9 +23,20 @@ const ProfileInfoContent: React.FC<ProfileInfoContentProps> = ({
   hideBio = false,
   wideFormat = false,
 }) => {
+  const currentUserFid = useDataStore((state) => {
+    const username = window?.localStorage.getItem('active_account');
```

Comment:
the active account is not stored in localStorage.
if an account is selected, it is stored in:
`AccountStore` which has `accounts` and `selectedAccountIdx`.
a common pattern to get it is: 
`const userFid = useAccountStore((state) => state.accounts[state.selectedAccountIdx]?.platformAccountId);`

Reply:
Thanks for quikly response, I will make change now.